larray/core/array.py

```diff
@@ -5637,6 +5638,8 @@ def to_hdf(self, filepath, key, *args, **kwargs):
         >>> a = ndtest((2, 3))
         >>> a.to_hdf('test.h5', 'a')  # doctest: +SKIP
         """
+        if isinstance(key, Group):
+            key = re.sub('[\\/?*\[\]:]', '_', str(_to_tick(key)))
```

Comment:
you reused the same invalid chars than for Excel, but I doubt the set of invalid chars is exactly the same.

Reply:
I just wanted to be consistent with `to_excel`.